pkgs/servers/sql/percona-server/8_0.nix

```
@@ -62,6 +62,11 @@ stdenv.mkDerivation (finalAttrs: {
   ] ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ rpcsvc-proto ];
 
   patches = [
+    # adapted from mysql80's llvm 19 fixes
+    ./libcpp-fixes.patch
+    # https://github.com/percona/percona-server/pull/5537
+    # https://github.com/Homebrew/homebrew-core/pull/204799
+    ./ps-9641.patch
```

```suggestion
      (fetchpatch {
        name = "fix-system-ssl-error.patch";
        url = "https://github.com/percona/percona-server/pull/5537/commits/a693e5d67abf6f27f5284c86361604babec529c6.patch";
        hash = "";
      })
```

How about using `fetchpatch` along with a brief comment describing what the patch fixes?

---

much better thank you 🙏